ui/src/environments/environment.stable.ts

```diff
@@ -12,7 +12,7 @@ export const environment: Environment = {
   gaId: 'UA-112406425-3',
   gaUserAgentDimension: 'dimension1',
   gaLoggedInDimension: 'dimension2',
-  zendeskHelpCenterUrl: 'https://aousupporthelp1580753096.zendesk.com/hc',
+  zendeskHelpCenterUrl: 'http://aousupporthelp.zendesk.com/hc',
```

Comment:
Do you need this? The link won't work as-is, since SSO is not configured for the stable gsuite to work with prod ZD. We might be able to update it to make that work though - would be nice if stable pointed to ZD prod.

Reply:
Ok, changed back. Switching this came up in our Slack convo but I didn't realize it wouldn't work as is.